99-make_reports.py

```diff
@@ -391,25 +398,13 @@ def run_report(subject, session=None):
 
     fname_report = bids_path.copy().update(suffix='report', extension='.html')
     rep.save(fname=fname_report, open_browser=False, overwrite=True)
+    import matplotlib.pyplot as plt  # nested import to help joblib
```

Comment:
We do have the same import at the top level already, is that intentional?

Reply:
I don't why I need to do this ... otherwise I have a crash with joblib :(

Reply:
But can you at least remove the top-level import then?

The crash also is something new, probably related to the Agg backend?